pkg/catalog/retry.go

```diff
@@ -0,0 +1,41 @@
+package catalog
+
+import (
+	"github.com/openservicemesh/osm/pkg/apis/policy/v1alpha1"
+	"github.com/openservicemesh/osm/pkg/identity"
+	"github.com/openservicemesh/osm/pkg/service"
+)
+
+// getRetryPolicy returns the RetryPolicySpec for the given downstream identity and upstream service
+// TODO: Add support for wildcard destinations
```

Comment:
Could we create a github issue for this please?

Reply:
Shashank added to the parent issue